impala/hiveserver2.py

```diff
@@ -1634,22 +1647,49 @@ def prettyprint_units(unit_val):
     def prettyprint_time(time_val):
         return prettyprint(time_val, ["ns", "us", "ms", "s"], 1000.0)
 
-    row = [label_prefix + node.label,
-           len(node.exec_stats),
-           prettyprint_time(avg_time),
-           prettyprint_time(max_stats.latency_ns),
-           prettyprint_units(cardinality),
-           prettyprint_units(est_stats.cardinality),
-           prettyprint_bytes(max_stats.memory_used),
-           prettyprint_bytes(est_stats.memory_used),
-           node.label_detail]
+    instances = 0
+    if node.exec_stats is not None:
+        instances = len(node.exec_stats)
+    latency = max_stats.latency_ns
+    cardinality_est = est_stats.cardinality
+    memory_used = max_stats.memory_used
+    memory_est = est_stats.memory_used
+    if (is_prettyprint):
+        avg_time = prettyprint_time(avg_time)
```

Comment:
probably it would be better to do it in another commit, but I think that formatting could be move to another file logic, e.g. exec_summary.py